stellargraph/layer/graph_attention.py

```diff
@@ -615,6 +619,18 @@ class GAT:
         normalize (str or None): normalization applied to the final output features of the GAT layers stack. Default is None.
         activations (list of str): list of activations applied to each layer's output; defaults to ['relu', ..., 'relu', 'linear'].
         saliency_map_support (bool): XXX What is this?
```

Comment:
If calculating saliency maps using the tools in `stellargraph.utils.saliency_maps` this should be True. Otherwise this should be False.